pkg/client/tx/client.go

```diff
@@ -203,7 +203,7 @@ func NewTxClient(
 // the synchronous error. If the function completes successfully, it returns an
 // either.AsyncError populated with the error channel which will receive if the
 // transaction results in an asynchronous error or times out.
-func (tClient *txClient) SignAndBroadcast(
+func (txClient *txClient) SignAndBroadcast(
```

Comment:
I think I see why we named it `tClient` before, some maybe `txnClient` is better.

Does this compile?